ex_8.13.c

```diff
@@ -0,0 +1,36 @@
+#include <stdio.h>
+#include <math.h>
+
+int main() {
+    int n, m;
+    printf("Enter n and m: \n");
+    scanf("%d %d", &n, &m);
+    int matrix1[n][m], matrix2[m][n], matrix3[n][n];

```

Comment:
This is C not C++ - is not allowed dynamic arrays this way

Reply:
Done!